docs/_static/llama-stack-spec.html

```diff
@@ -3626,7 +3626,8 @@
                         "type": "string"
                     },
                     "enable_session_persistence": {
-                        "type": "boolean"
```

Comment:
Just checking -- did you update this using `cd docs/openapi_generator && sh run_openapi_generator.sh`?

Reply:
Yes

Reply:
I removed the unrelated change (accidentally included by my other work)